src/coreclr/src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -438,6 +458,46 @@ public bool NonTrivialInterfaceCast
             }
         }
 
+        public bool HasDefaultConstructor
+        {
+            get
+            {
+                return (Flags & enum_flag_HasDefaultCtor) != 0;
+            }
+        }
+
+        public bool HasClassConstructor
```

Comment:
This property and the related backing field should be removed - they're not needed as of the latest iteration.